vcd/resource_vcd_vapp_vm_hot_updates_test.go

```diff
@@ -25,15 +25,17 @@ func TestAccVcdVAppHotUpdateVm(t *testing.T) {
 	}
 
 	var params = StringMap{
-		"Org":         testConfig.VCD.Org,
-		"Vdc":         testConfig.VCD.Vdc,
-		"EdgeGateway": testConfig.Networking.EdgeGateway,
-		"Catalog":     testSuiteCatalogName,
-		"CatalogItem": testSuiteCatalogOVAItem,
-		"VAppName":    hotVappName,
-		"VMName":      hotVmName1,
-		"Tags":        "vapp vm",
-		"Media":       testConfig.Media.MediaName,
+		"Org":             testConfig.VCD.Org,
+		"Vdc":             testConfig.VCD.Vdc,
+		"EdgeGateway":     testConfig.Networking.EdgeGateway,
+		"Catalog":         testSuiteCatalogName,
+		"CatalogItem":     testSuiteCatalogOVAItem,
+		"VAppName":        hotVappName,
+		"VMName":          hotVmName1,
+		"Tags":            "vapp vm",
+		"Media":           testConfig.Media.MediaName,
+		"StorageProfile1": testConfig.VCD.ProviderVdc.StorageProfile1,
```

Comment:
A bit hard to see in the test, but is there a case where One step has storage profile defined, but the next step removes value at all?

Reply:
no, there isn't such step.

Reply:
The question is - should we have such step then?

Reply:
you cant' remove storage profile for VM

Reply:
also, this field is computed and optional - as if user didn't provide value then API uses default one. As result if remove storage profile, no changes will be done.